snippets/facets.liquid

```diff
@@ -115,14 +115,15 @@
             {%- for filter in results.filters -%}
               {% liquid
                 assign total_active_values = total_active_values | plus: filter.active_values.size
-                case filter.presentation
-                  when 'swatch'
-                    assign show_more_number = 15
-                    assign visual_layout_class = 'facets-layout-grid facets-layout-grid--' | append: filter.presentation
-                  else
-                    assign show_more_number = 10
-                    assign visual_layout_class = 'facets-layout-list'
-                endcase
+                assign presentation = filter.presentation | default: 'text'
```

Comment:
Not sure I get what you mean. Unfortunately, as of now, rendering filters is duplicative. What exactly do you mean by "have the default presentation value defined somewhere it could be used in every place we're doing"?

Reply:
Oh gotcha that makes more sense. I've made the change